autoload/android.vim

```diff
@@ -248,9 +248,10 @@ function! android#setClassPath()
     return
   endif
 
-  if exists(":JCstart")
```

Comment:
The reason for this conditional is checking for JavaComplete2, which will manage classpaths for us. Do you have a good reason to remove it?